packages/falcon-sensor/charts/templates/NOTES.txt

```diff
@@ -0,0 +1,12 @@
+You should be a Crowdstrike customer and have access to the Falcon Linux Sensor
+and Falcon Container Downloads to install this helm chart and have it work
+correctly as a specialized container has to exist in the container registry
+before this chart will install properly.
+
+The CrowdStrike Falcon sensor should be spinning up on all your kubernetes nodes
+now. There should be no further action on your part unless no Falcon Sensor
+container exists in your registry. If you forgot to add a Falcon Sensor image to
+your image registry before you ran `helm install`, please add the Falcon Sensor
+now; otherwise, pods will fail with errors and crash until there is a valid
+image to pull. The default image name to deploy a kernel sensor to a node is
+`falcon-node-sensor`.
```

Comment:
```suggestion
Access to the Falcon Linux Sensor and Falcon Container Downloads [1] is
required to install this Helm chart. This is provided automatically to all active CrowdStrike customers.

Additionally, a containerized sensor must be present in a container registry accessible from your Rancher installation. Sample Dockerfiles are available at [2].

CrowdStrike Falcon sensors will deploy across all nodes in your Rancher cluster after
running this Helm chart. If errors are seen, an extremely common error is accidentally
forgetting to add your containerized sensor to your local image registry prior to executing
`helm install`. The default image name to deploy a kernel sensor to a node is `falcon-node-sensor`.

[1] https://falcon.crowdstrike.com/hosts/sensor-downloads
[2] https://github.com/CrowdStrike/Dockerfiles
```

Reply:
@shawndwells can you please open a ticket and address this upstream?

Reply:
This has been fixed upstream